internal/cosmosdb/cosmosdb_client.go

```diff
@@ -0,0 +1,28 @@
+package cosmosdb
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"log"
+
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+	utils "pagopa.it/pagopa-payment-wallet-helpdesk-service/internal/utils"
+)
+
+const MongoConnectionString = "mongodb://${MONGO_USERNAME}:${MONGO_PASSWORD}@${MONGO_HOSTNAME}:${MONGO_PORT}/${MONGO_DB_NAME}?ssl=${MONGO_SSL_ENABLED}"
+
+func GetMongoClient() (*mongo.Client, error) {
+	uri, err := utils.ReplaceEnvPropertiesInString(MongoConnectionString)
+	if err != nil {
+		errorMessage := fmt.Sprintf("Cannot instantiate Mongo client, error building connection string: [%s]", err)
+		log.Fatal(errorMessage)
```

Comment:
[This kills the server](https://pkg.go.dev/log#Fatal), maybe it's better to let the caller handle errors?

Reply:
Yes, you're right, fixing it

Reply:
fixed with https://github.com/pagopa/pagopa-payment-wallet-helpdesk-service/pull/5/commits/f9c6d2ef25bb25ee1729f27267002c54000897b3